WPMedia/arc-themes-components
# Add formatSocialURL

## Ticket

- [TMEDIA-717](https://arcpublishing.atlassian.net/browse/TMEDIA-717)
- [TMEDIA-758](https://arcpublishing.atlassian.net/browse/TMEDIA-758)

## Description

Moving formatSocialULR into components util

## Acceptance Criteria

n/a

## Test Steps

1. Checkout branch - `git checkout add-format-social-url`
2. Run Storybook `npm run storybook`
4. Check out the button storybook documentation

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
